grammars/asciidoc.cson

```diff
@@ -48,7 +48,7 @@
   # ========================
   #
   {
-    'begin': '^\\.(\\w[\\w ]*\\w)'
+    'begin': '^\\.(\\w.*\\w)'
```

Comment:
To be accurate, the first character should be matched as:

```
[^\s.]
```

Which means "not space or period".

There is no need to match a trailing word character. Thus, the whole thing can be shorted to:

```
'begin': '^\\.([^\\s.].*)'
```